deployments/chef/metadata.rb

```diff
@@ -3,13 +3,15 @@
 maintainer_email 'support@signalfx.com'
 license 'Apache-2.0'
 description 'Installs/Configures the SignalFx Agent'
-version '0.2.6'
+version '0.2.7'
```

Comment:
I would go to 0.3.0, this is a pretty big addition.

Reply:
Also don't do the release increment until right before release.

Reply:
Reverted back to 0.2.6. Will increment to 0.3.0 when releasing to chef supermarket.